README.md

```diff
@@ -26,31 +27,28 @@ planx-new is a monorepo containing our full application stack. Here's a quick su
 
 1. Clone this repository. You'll need to get some additional environment variable files that aren't published here. Find instructions for copying these in 1password under the AWS Staging IAM user role entry
 
-1. Run the following command to get everything (postgres, sharedb, api and hasura server processes) up and running `docker-compose up --build -d`
-
-1. Move into the hasura directory `cd ../hasura.planx.uk` & install dependencies `pnpm i`. 
+1. Run `pnpm docker:up` from the project root to get everything (postgres, sharedb, api and hasura server processes) up and running and `pnpm docker:down` to stop all services. `pnpm docker:down-hard` will remove volumes (i.e. database data) and can be a useful hard reset when necessary.
 
-1. Open [Hasura's](https://hasura.io/) web console `pnpm start` & check that your Google email address is in the `users` table, if not then add it
+1. Open [Hasura's](https://hasura.io/) web console `http://localhost:7000` & check that your Google email address is in the `users` table, if not then add it
```

Comment:
should this be localhost:9695 for Hasura? or if it's changed to 7000 now, could env.example `HASURA_CONSOLE_PORT=9695` be updated please?